### DISCUSSION TITLE
Agent system startup (back online) as a trigger / schedule option

### DISCUSSION THREAD

**Discussion by mbr-a** [amidaware/tacticalrmm#1879] [Ideas]
It can be usefull to have a trigger when an agent (computer) is started and back online.
Initialy my need is the capability to have a notification (email ?) when a specific / targeted computer is back from an offline state.
But after more related usecase it can be usefull also to have this kind of schedule capability for automation manager.

**Comment by dinger1986** [comment]
automation manager can do this with an onboarding task (so it will run a task the first time a machine contacts it since the task has been added).

I setup a check for a service that doesnt exist or a ping to an ip that doesnt exist and when that is triggered know the machine is online, a bit janky but it works

**Reply by mbr-a** [reply]
Thanks for the trick !

**Comment by wh1te909** [comment]
If you have overdue alerts enabled for an agent, you'll automatically get an email when the agent goes offline and another email when it comes back online. You can either set this up individually per agent (by checking the checkboxes next to each agent in the main dashboard) or through a policy with Alert Manager.

![chrome_BvkQ5pJbaT](https://github.com/amidaware/tacticalrmm/assets/7434746/afd79d7d-eeaf-4659-9e0e-a94e5c9f8ccf)
